src/JsonHelper.ts

```diff
@@ -1,17 +1,42 @@
-// @ts-nocheck
-import { Schema, SchemaHelper, ConfluentSubject, ConfluentSchema } from './@types'
+import {
+  Schema,
+  SchemaHelper,
+  ConfluentSubject,
+  ConfluentSchema,
+  SchemaResponse,
+  SchemaType,
+  ReferenceType,
+  JsonConfluentSchema,
+  ProtocolOptions,
+} from './@types'
 import { ConfluentSchemaRegistryError } from './errors'
 
 export default class JsonHelper implements SchemaHelper {
-  public validate(schema: Schema): void {
+  public validate(_schema: Schema): void {
     return
   }
 
   public getSubject(
-    confluentSchema: ConfluentSchema,
-    schema: Schema,
-    separator: string,
+    _confluentSchema: ConfluentSchema,
+    _schema: Schema,
+    _separator: string,
```

Comment:
same here, consistency with `SchemaHelper`

Reply:
I agree - but this is how TS was set up. It marks it as a problem if unused, to fix it add underscore.